pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
```

Comment:
I don't think it's customary in nixpkgs to use two lines of whitespace like this.

Reply:
Point taken, I'll remove them.